1-exercises/D-for-in-loops/exercise-2.js

```diff
@@ -25,6 +25,10 @@ let shapesWithMoreThanFiveSides = [];
 
 // WRITE YOUR CODE BELOW THIS LINE
 
+for (const key in shapes) {
+if (shapes[key] > 5) console.log(`${key}: ${shapes[key]}`);
```

Comment:
This exercise is different from the previous one. In this case, you need to create an array and console.log it. Please read the instructions.